src/Task.gren

```diff
@@ -125,6 +125,32 @@ map func taskA =
     taskA
         |> andThen (\a -> succeed (func a))
 
+{-| Apply the results of two tasks together if they succeed. Both tasks are run concurrently, and if either task fails the whole task fails:
+
+    succeed ((+) 2)
+        |> andMap (succeed 3)
+    --> succeed 5
+
+    fail "oh dear"
+        |> andMap (succeed 3)
+    --> fail "oh dear"
+
+This can be used to do `Task.mapN` for any number of arguments:
```

Comment:
Maybe add an explicit line that this can be helpful when `map2` or `map3` isn't enough?